webapp/channels/src/components/widgets/inputs/input/input.tsx

```diff
@@ -116,6 +133,28 @@ const Input = React.forwardRef((
     const handleOnChange = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
         setCustomInputLabel(null);
 
+        const newValue = event.target.value;
+
+        // Check for maxLength (limit) validation in real-time
```

Comment:
Same comment here as with the `AdvancedTextbox` widget regarding checking the length while rendering instead of storing that in state

Reply:
ok, using derived value intead.